category.php

```diff
@@ -1,12 +1,8 @@
 <?php get_header(); ?>
 
-<div class="hero-container">
-  <div class="hero-image" style="background-image: url(<?php echo get_template_directory_uri(); ?>/images/bg/contact.jpg);">
-    <div class="container">
-      <div class="hero-heading">
-        <h1> Projects </h1>
-      </div>
-    </div>
+<div class="hero">
```

Comment:
I don't think a partial is a good idea because too many of the pages will require customized h1 titles rather than just the wp_title